src/shared/l10n/locales/da.json

```diff
@@ -0,0 +1,174 @@
+        "reportsDataDirectory": "Rapport Datamappe",
+        "reportsDataDirectoryButton": "Gennemse",
+        "reportsDataDirectoryTip": "(hvor Ace gemmer rapporter internt)",
+        "save": "Gem",
+        "title": "Ace Indstillinger",
+        "userInterfaceLanguage": "Sprog",
+        "userInterfaceLanguageTip": "(vælg sprog til grænsefladens tekst og beskeder)"
+    },
+    "report": {
+        "images": "Billeder",
+        "imagesSection": {
+            "NA": "N/A",
+            "altAttribute": "'alt' attribut",
+            "ariaDescribedbyContent": "'aria-describedby' indhold",
+            "associatedFigcaption": "Tilhørende 'figcaption'",
+            "image": "Billede",
+            "location": "Placering",
+            "noImages": "Ingen billeder fundet i publikationen.",
+            "role": "Rolle"
+        },
+        "metadata": "Metadata",
+        "metadataSection": {
+            "a11y": "Tilgængelighed",
+            "allPresent": "Alle krævede metadata felter om tilgængelighed er til stede.",
+            "missing": "Manglende tilgængeligheds Metadata:",
+            "name": "Navn",
+            "value": "Værdi"
+        },
+        "outlines": "Struktur",
+        "outlinesSection": {
+            "headings": "Overskrift struktur",
+            "html": "HTML struktur",
+            "toc": "EPUB Indholdsfortegnelse"
+        },
+        "summary": "Oversigt",
+        "summarySection": {
+            "best-practice": "Best Practice",
+            "critical": "Kritisk",
+            "minor": "Mindre",
+            "moderate": "Moderat",
+            "other": "Andre",
+            "serious": "Stor",
+            "total": "Total",
+            "type": "Type"
+        },
+        "title": "Rapport",
+        "violations": "Overtrædelser",
+        "violationsSection": {
+            "details": "Detaljer",
+            "impact": "Betydning",
+            "learnAbout": "Læs om:",
+            "location": "Placering",
+            "noViolations": "Ingen overtrædelser rapporteret.",
+            "rule": "Regel",
+            "ruleset": "Regelsæt",
+            "via": "Via:"
+        }
+    },
+    "sidebar": {
+        "checkEPUB": "Tjek EPUB",
+        "export": "Eksportér",
+        "history": "Historik",
+        "maximize": "Maksimér Sidebar",
+        "minimize": "Minimér Sidebar",
+        "reRun": "Kør igen",
+        "settings": "Indstillinger"
+    },
+    "splash": {
+        "browseForFileOrFolder": "søg efter en __fil__ eller en __mappe__ .",
+        "clickToBrowse": "klik her for at gennemse.",
+        "dropHere": "Træk en EPUB fil eller mappe ind her,",
+        "or": "eller",
+        "orDropSidebar": "eller på __ knappen i sidebaren,",
+        "title": "Ace, af DAISY"
+    },
+    "tablePagination": {
+        "firstPage": "Første Side",
+        "lastPage": "Sidste Side",
+        "nextPage": "Næste Side",
+        "previousPage": "Forrige Side"
```

Comment:
@ddfreiling lines 163,164 and165 ends with a comma, then shouldn't line 165 too?

Reply:
Not in JSON, comma is only between adjacent properites.